react/features/base/dialog/components/StatelessDialog.web.js

```diff
@@ -112,36 +125,86 @@ class StatelessDialog extends Component<Props> {
      * @returns {ReactElement}
      */
     render() {
-        return (
+        const {
+            children,
+            t /* The following fixes a flow error: */ = _.identity,
+            titleString,
+            titleKey,
+            width
+        } = this.props;
 
-            /**
-             * Enabled light theme for dialogs until all in-dialog components
-             * support dark theme.
-             */
-            <AtlasKitThemeProvider mode = 'light'>
```

Comment:
@virtuacoplenny I think the device selection dialog for filmstrip only mode is not looking correct. Maybe is related to this change.

Reply:
Ignore this! It is working!